src/blocks/collapsible-control/frontend.js

```diff
@@ -58,7 +58,7 @@ bu_blocks.collapsibleControl = ( function() {
 	 * Find all Collapsible blocks on a page
 	 */
 	var findAllCollapsibleBlocks = function() {
-		var containers = document.querySelectorAll( '.wp-block-bu-collapsible' );
+		var containers = document.querySelectorAll( '.js-wp-block-bu-collapsible' );
```

Comment:
Derp - I made all my comments in the dist version of this. Same idea applies, though.

Does `docs` need to have a copy of this file? It looks like it might. Could you import this to a file in docs instead? Or maybe that's another compiled folder - if so, please disregard this comment. :)

Reply:
the docs folder should have a copy of the compiled files. That gets copied in via grunt